bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/profiles/SystemProfiles.java

```diff
@@ -37,7 +37,7 @@
     StateProfileType FOLLOW_TYPE = ProfileTypeBuilder.newState(FOLLOW, "Follow").build();
 
     StateProfileType OFFSET_TYPE = ProfileTypeBuilder.newState(OFFSET, "Offset").withSupportedItemTypes("Number")
-            .build();
+            .withSupportedItemTypesOfChannel("Number").build();
```

Comment:
Btw, would be nice if you could use the constant `CoreItemFactory.NUMBER`.´(also in the call of withSupportedItemTypes).

Reply:
Ah we have constants for the item types, nice. Thanks for pointing this out to me. I have added a commit that addresses it.